src/guides/v2.3/frontend-dev-guide/theme-best-practice.md

```diff
@@ -30,5 +30,10 @@ We recommend using the following best practices when developing themes:
    ```xml
    <a href="#"><?= __('Click to download'); ?></a>
    ```
+1. By following the standard conventions, you give your code a professional look while making it easier to read. Ensure that your Magento coding standards are based on PSR coding standards.
+1. When inheriting Blank or Luma themes, Make use of the mobile first approach that is already been followeed in these themes.
```

Comment:
1. When inheriting `blank` or `luma` themes. Make use of the mobile first approach that is already been followed in these themes.